Cargo.toml

```diff
@@ -8,7 +8,8 @@ members = [
     ## Middleware
     "middleware/template",
     "middleware/diesel",
-    "middleware/jwt",
+    # TODO: Re-enable middleware when their dependencies are updated
+    # "middleware/jwt",
```

Comment:
If this is going to be merged with these commented out, can we get issues for each one which needs updating? I know there are notes in this PR, but that's not as simple for a beginner to pick up, should they choose to.

Reply:
Do you want me to create these issues?  I'm not sure of the future of these middleware given the state of the dependencies.